modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImplTest.java

```diff
@@ -607,7 +633,8 @@ protected TestTable createTable(String name, int size, IgniteDistribution distr,
                 new TableDescriptorImpl(columns, distr),
                 name,
                 ColocationGroup.forNodes(nodeNames),
-                size
+                size,
+                () -> mappingException
```

Comment:
```suggestion
                size
```
Let's just override the `colocationGroup` method instead, and add a short explanation on why we throws an exception from here.

Reply:
Thanks, I've fixed it as per your suggestion.